packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -194,7 +201,15 @@ const main = () => {
                 'immediately deploy the bundle to this target once it is pushed'
             )
         )
-        .action(({buildDirectory, message, projectSlug, target}) => {
+        .addOption(
+            new program.Option(
+                '-c, --credentialsFile <credentialsFile>',
+                'the file where your credentials should be saved'
```

Comment:
Looks like you missed updating the _option_ text here. Should probably be something along the lines of "the location of your credentials file".

Reply:
Isn't this the second option the `'the file where your credentials should be saved'`?

Reply:
I'm not sure I understand you comment. It looks like here you are adding an option to the "push" command, the option you are adding is the ability to use a credential files from a given location. Am I wrong in suggesting that this options description is not the same as the option that you added to the "save-credentials" command? I think this was simply a copy past error.

Reply:
Oh. I see now. I misunderstood your comment. Is it ok now?

Reply:
Thanks Daniel, that looks good. 
I'm just going to have another member on the team take a look at the PR to be doubly sure things look good and I'll be that extra step forward. 👍